addons/rook/template/base/install.sh

```diff
@@ -520,16 +514,6 @@ function rook_should_fail_install() {
     return 1
 }
 
-function rook_maybe_bluefs_buffered_io() {
-    local dst="${DIR}/kustomize/rook/operator"
-
-    semverParse "$ROOK_VERSION"
-    local rook_major_minor_version="${major}.${minor}"
-    if [ "$rook_major_minor_version" = "1.8" ] || [ "$rook_major_minor_version" = "1.9" ]; then
-        sed -i "/\[global\].*/a\    bluefs_buffered_io = false" "$dst/configmap-rook-config-override.yaml"
-    fi
-}
-
```

Comment:
Why are we removing this? Wouldn't we still want this if upgrading _to_ Rook 1.8 or 1.9 from 1.4?

Reply:
Same here: https://github.com/replicatedhq/kURL/pull/3760/files#r1036420648 :-P

Reply:
this is in the template used to generate new versions. it is [still](https://github.com/replicatedhq/kURL/blob/6737ccb509692ba58d74fe1dc45482aee579242c/addons/rook/1.9.12/install.sh#L172) in the rook add-on versions affected by the issue. if a user were to upgrade to a later version they would no longer be on a version of ceph affected by the issue (17 quincy).

Reply:
ah, that's right! The affected 1.8 and 1.9 versions will still have this fix but not this version and later. Got it.